colossalai/fx/passes/algorithms/ckpt_solver_chen.py

```diff
@@ -73,10 +73,11 @@ def run_chen_greedy(b: int = 0) -> Tuple[Set, int]:
         y = 0
         prev_idx = 2
         for (idx, n) in enumerate(gm.graph.nodes):
-            temp += getattr(n, 'fwd_out')
+            n: Node
+            temp += n.meta['fwd_mem_out'] + n.meta['fwd_mem_tmp']
             y = max(y, temp)
             if temp > b and n in ckpt_nodes:
-                x += getattr(n, 'fwd_out')
+                x += n.meta['fwd_mem_out']
```

Comment:
Now `fwd_mem_out` are stored in `node.meta`.

Reply:
should it be `n.meta.fwd_mem_out`?

Reply:
`node.meta` is a `dict` in the FX definition.

Reply:
what is fx definition?

Reply:
The PyTorch fx.Node. `torch.fx.Node().meta`

Reply:
Ok, i saw it in the source code.